sdk/eventhub/azure-eventhub-checkpointstoreblob/azure/eventhub/extensions/checkpointstoreblob/_blobstoragecs.py

```diff
@@ -265,8 +284,7 @@ def claim_ownership(self, ownership_list, **kwargs):
                 pass
         return gathered_results
 
-    def update_checkpoint(self, checkpoint, **kwargs):
-        # type: (Dict[str, Optional[Union[str, int]]], Any) -> None
+    def update_checkpoint(self, checkpoint: Dict[str, Any], **kwargs: Any) -> None:
```

Comment:
given that we know the contents of the dict, can we narrow down the Any ? union[str, any] or perhaps a typeddict ?